core/src/solvers/pipeline_planner.cpp

```diff
@@ -208,11 +220,10 @@ PlannerInterface::Result PipelinePlanner::plan(const planning_scene::PlanningSce
 			// Choose the first solution trajectory as response
 			result = solution.trajectory;
 			last_successful_planner_ = solution.planner_id;
-			return { true, "" };
+			return MoveItErrorCode(solution.error_code.val, solution.error_code.message, solution.error_code.source);
 		}
```

Comment:
```suggestion
		}
			return MoveItErrorCode(solution.error_code.val, solution.error_code.message, solution.error_code.source);		
```
As pointed out above, the complete error code should be returned in any case